slave/runners.py

```diff
@@ -41,14 +43,16 @@ def __init__(self, info):
 
     def rm(self, path):
         if not os.path.exists(path):
-            print "rm", path, "(non-existing)"
+            logger = logging.getLogger('Runner.rm')
```

Comment:
Maybe there could be only one logger belonging to the class instance, created in the ctor instead?

Reply:
We don't need the resolution for the logger names that I originally used.  I'll change it for any class where we are adding logging calls. Thanks. I wasn't happy about it but left it as it was.